Source/JavaScriptCore/b3/B3ArgumentRegValue.cpp

```diff
@@ -30,9 +30,7 @@
 
 namespace JSC { namespace B3 {
 
-ArgumentRegValue::~ArgumentRegValue()
-{
-}
+ArgumentRegValue::~ArgumentRegValue() = default;
```

Comment:
Shouldn't we use default directly at the header files?

Reply:
I have seen `= default` used in WebKit code in this way, can you share example, so I can approach it your suggested way.

Reply:
Yeah, I think it is a style thing. To be honest I don't know what is our style regarding this but that's the first time I've seen it written that way.

Example: https://github.com/WebKit/WebKit/blob/78d8c59ed298b1c74189daf439a9104dbb7b2e2c/Source/WTF/wtf/HashMap.h#L89

Reply:
Ahmad, please do not apply this suggestion. We do not want to inline constructors / destructors in this patch, this would not be identical to the pre-existing code and could have performance implications.